test/sql/create_hypertable.sql

```diff
@@ -11,6 +11,7 @@ select create_hypertable('test_schema.test_1dim', 'time');
 
 \dt "test_schema".*
 
+insert into test_schema.test_1dim VALUES ('2004-10-19 10:23:54+02',1.0);
```

Comment:
Can you actually do this afterward the create_hypertable call (and then have a second create_hypertable call)? It'd be nice to test both cases: (1) where the target hypertable is empty and (2) where the target hypertable is not empty.